deploy/crds/apps.open-cluster-management.io_subscriptions_crd_v1.yaml

```diff
@@ -123,8 +156,8 @@ spec:
                       type: string
                     clusterOverrides:
                       items:
+                        description: ClusterOverride describes rules for override
                         type: object
-                        x-kubernetes-preserve-unknown-fields: true
```

Comment:
Thanks XJ. I went with the safest option which is updating the existing CRD with specific changes instead of using the generated CRD.

I haven't problems access my clusters so the PR is on hold until I've verified the changes.